_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -3088,6 +3284,26 @@ public static function validate_string( $value, $request, $param ) {
 		return true;
 	}
 
+	/**
+	 * Validates that the parameter is an array of strings.
+	 *
+	 * @param array           $value Value to check.
+	 * @param WP_REST_Request $request The request sent to the WP REST API.
+	 * @param string          $param Name of the parameter passed to the endpoint holding $value.
+	 *
+	 * @return bool|WP_Error
+	 */
+	public static function validate_array( $value, $request, $param ) {
```

Comment:
I wonder if we should change the method's name to make it clearer that it doesn't just check that we're dealing with an array, to avoid confusion in the future for others looking to do that for other endpoints. Maybe something like `validate_array_of_strings`? What do you think?

Reply:
Good idea, done!